src/helpers/firebaseHelpers.js

```diff
@@ -10,6 +10,30 @@ export const parseData = (querySnapshot) =>
       {
         id,
         ...product,
+        /***
+         * We most transform the data to have acces
+         * to a propper aria-label and it's color
+         */
+        timeFrameLabel: nextPurchaseDay(
+          product.daysUntilNextPurchase,
+          product.lastPurchaseDate,
+          product.numberOfPurchases,
+        ),
       },
-    ];
+    ].sort((a, b) => {
+      /***
+       * Firt we verified if days are equals in order to sort names alphabetically
+       * If not we sort the items by the days
+       */
```

Comment:
I'm not sure but about your sorting logic: 
**How are you sorting the `inactive` products?**

I guess it could be better if you might use the `timeFrameLabel` property because you're processing some information and you're getting `soon`, `kindOfSoon`, `notAnySoon`, and `inactive` too.

What do you think about it? or Am I wrong?

Reply:
Never mind. All of this! 😄